### DISCUSSION TITLE
share to: (media) open in graysky

### DISCUSSION THREAD

**Discussion by ghost** [mozzius/graysky#80] [Feature requests]
On ios/android, when looking at a photo gallery, if i press the built in share button it has the option to share media via message, airdrop, email, post to social media, I would love the ability to see greysky here

![IMG_0812](https://github.com/mozzius/graysky/assets/66647238/d4b645eb-d680-47a5-a00d-d5b337d1c412)

**Comment by mozzius** [comment]
Share intent is tricky to get working in Expo, but yes I'd love to see this!
